platform/ios/MGLMapView.mm

```diff
@@ -2098,6 +2098,36 @@ - (void)removeAnnotations:(NS_ARRAY_OF(id <MGLAnnotation>) *)annotations
     _mbglMap->removeAnnotations(annotationIDsToRemove);
 }
 
+- (void)addOverlay:(id <MGLOverlay>)overlay
+{
+    [self addOverlays:@[ overlay ]];
+}
+
+- (void)addOverlays:(NS_ARRAY_OF(id <MGLOverlay>) *)overlays
+{
+    for (id <MGLOverlay> overlay in overlays)
+    {
+        assert([overlay conformsToProtocol:@protocol(MGLOverlay)]);
+    }
+
+    [self addAnnotations:overlays];
+}
+
+- (void)removeOverlay:(id <MGLOverlay>)overlay
+{
+    [self removeOverlays:@[ overlay ]];
+}
+
+- (void)removeOverlays:(NS_ARRAY_OF(id <MGLOverlay>) *)overlays
+{
+    for (id <MGLOverlay> overlay in overlays)
+    {
+        assert([overlay conformsToProtocol:@protocol(MGLOverlay)]);
```

Comment:
This should be an `NSAssert()` to work with Objective-C exception breakpoints (or breakpoints on `-[NSException raise]`).